files/usr/share/cinnamon/applets/power@cinnamon.org/applet.js

```diff
@@ -366,12 +366,29 @@ MyApplet.prototype = {
                         // Info for the primary battery (either the primary device, or any battery device if there is no primary device)
                         if (device_type == UPDeviceType.BATTERY && !showed_panel_info) {
                             this.set_applet_tooltip(status);
-                            if (this.showpercentage) {
-                                this.set_applet_label("%d%%".format(Math.round(percentage)));
+                            let labelText = "";
+                            if (this.labelinfo == "nothing") {
+                                ;
                             }
-                            else {
-                                this.set_applet_label("");
+                            else if (this.labelinfo == "time" && seconds != 0) {
+                                let time = Math.round(seconds / 60);
+                                let minutes = Math.floor(time % 60);
+                                let hours = Math.floor(time / 60);
+                                labelText = _("time of battery remaining", "%d:%02d").format(hours,minutes);
```

Comment:
You're wrong with the use of `_(msgid)`, you want to use `C_(msgctx, msgid)` (it point to `Gettext.pgettext`)